### ISSUE TITLE
Camvid dataset labels

### ISSUE THREAD

**Issue by QingXiaoYan** [qubvel-org/segmentation_models.pytorch#1072]
It seems that there are issues with the annotations in the Camvid dataset. No matter how I train it, the IoU for some categories is particularly low.

![Image](https://github.com/user-attachments/assets/b9adaeac-2501-416d-bfc2-b9e874e79c59)

**Comment by qubvel** [comment]
Hey @QingXiaoYan, indeed, that's might be the case, I would recommend to visualize predictions and ground truth for the low score classes to inspect it.

**Comment by QingXiaoYan** [comment]
OK,thank you
